site/faq.md

```diff
@@ -99,6 +99,13 @@ explanation.
 Flux needs a deploy key to be allowed to push to the version control
 system in order to read and update the manifests.
 
+
+### What is auto-matically synced back to git by Flux?
```

Comment:
It may be clearer to answer a more specific, yes/no question. I think the original misdirection was in the sentence above: 

> Flux needs a deploy key to be allowed to push to the version control system in order to read and update the manifests

so the follow-up question could be "Will Flux update git when I change the cluster?", with an unambiguous, unqualified answer.